lib/suite.js

```diff
@@ -62,6 +62,7 @@ function Suite(title, parentContext, isRoot) {
   this.suites = [];
   this.tests = [];
   this.pending = false;
+  this.skipped = false;
```

Comment:
yeah, so, if it's not possible to have `this.pending && this.skipped === true`, then we should use a `state`